docs/sources/structure/topic-types/tutorial/index.md

```diff
@@ -114,7 +114,7 @@ Depending on the application, your tutorial's data might be:
 - In demo repositories that the user clones locally
 
 For example, the [Play with Grafana Mimir](https://grafana.com/tutorials/play-with-grafana-mimir/) tutorial provides a repository that users can clone to complete the tutorial.
-As a comparison, the [Store exemplars in Grafana Mimir](https://grafana.com/docs/mimir/latest/operators-guide/use-exemplars/storing-exemplars/) topic is a pure task that a user would follow to complete their work.
+As a comparison, the [Store exemplars in Grafana Mimir](https://grafana.com/docs/mimir/latest/manage/use-exemplars/store-exemplars/) topic is a pure task that a user would follow to complete their work.
```

Comment:
This topic goes to great lengths to explain the distinction between a tutorial and a task-oriented topic (what is commonly referred to as a how-to guide). Then why does this sentence tell us that a tutorial can be composed purely of a task? This is confusing.

Reply:
I think it is saying that _Store exemplars in Grafana Mimir_ is a task topic and not a tutorial.

Reply:
In that case, it's a contrast not a comparison. I have added an appropriate suggestion.

Reply:
Makes sense! Thanks for the suggestion, please re-review and remove the request for changes if you are happy :)